packages/terra-form-fieldset/src/Fieldset.scss

```diff
@@ -0,0 +1,41 @@
+:local {
+  .fieldset {
+    border: 0;
+    margin-bottom: 1.428em;
```

Comment:
Should this be themeable?

Reply:
Discussed with @bjankord about theming margins and  we decided not to theme margins untill stated explicitly in the callouts (which we don't have now).